packages/backend-common/src/reading/GithubUrlReader.ts

```diff
@@ -92,13 +92,31 @@ export class GithubUrlReader implements UrlReader {
     return response.buffer();
   }
 
+  private getCredentials = async (
+    url: string,
+    options?: { token?: string },
+  ): Promise<GithubCredentials> => {
+    if (options?.token) {
```

Comment:
Thinking a bit about what to do if a token is passed by ignored. It could that the best thing to do for now is to update all other readers to throw an error if a token is provided? Something along the lines of "user token not supported when reading from ...". I could see that backfiring for example for templates that support publishing to different targets though, so could be that that's gonna cause more trouble than it's worth.